x509util/marshal_utils.go

```diff
@@ -129,6 +130,44 @@ func (m *MultiURL) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// MultiOID is a type used to unmarshal a JSON string or an array
+// of strings into a []x509.OID.
+type MultiOID []x509.OID
+
+// MarshalJSON implements the json.Marshaler interface for MultiOID.
+func (m MultiOID) MarshalJSON() ([]byte, error) {
```

Comment:
This and `UnmarshalJSON` may need a test

Reply:
Fixed with 67ff3ee, I've also added tests with more complex policy templates.